scripts/pythonContractConverter.py

```diff
@@ -0,0 +1,115 @@
+# assumes all return types are strings
+def parseContract(file, funcName):
```

Comment:
Would add type hints then
```suggestion
# assumes all return types are strings
def parseContract(file: str, funcName: str) -> str:
```

Reply:
I fully agree that type-hints are a good practice, and make the commented assumption at least parsed (even if not enforced).

I might additionally suggest removing the comment if the type hints are added.

Reply:
This actually refers to the return types of functions which are parsed by `parseContract`, so this behavior cannot be represented as a type hint (or at least not a standard Pythonic type hint). That said, these type hints are correct and could be added without an issue in this context, if helpful.